src/util.rs

```diff
@@ -355,9 +359,7 @@ pub(crate) fn get_adapter_mtu(luid: &NET_LUID_LH) -> std::io::Result<usize> {
             }
         }
 
-        if let Err(e) = FreeMibTable(if_table as *mut _) {
-            log::trace!("Failed to free MIB table: {}", e);
-        }
+        FreeMibTable(if_table as *mut _);
```

Comment:
```suggestion
        if let Err(e) = FreeMibTable(if_table as *mut _) {
            log::trace!("Failed to free MIB table: {}", e);
        }
```
Why remove the logging on error?